Packs/Code42/Integrations/Code42/Code42.py

```diff
@@ -1021,9 +1018,6 @@ def securitydata_search_command(client, args):
 
 @logger
 def file_events_search_command(client, args):
-    file_events_version = demisto.incident()["CustomFields"].get("code42fileeventsversion", "1")
-    if file_events_version != "2":
```

Comment:
I had initially added those error conditions to prevent users from mixing the different event formats within the same incident (e.g. an incident fetched with V1 events shouldn't be able to make a V2 event search and add them to the V1 incident), but I realized after the release that this prevents running these commands from scheduled job playbooks (or testing in playbooks) without first manually setting the incident context value to the correct file event version.